pylib/cqlshlib/test/test_cqlsh_completion.py

```diff
@@ -1048,12 +1048,12 @@ def test_complete_in_use(self):
                                              'system_virtual_schema', 'system_cluster_metadata'])
 
     def test_complete_in_create_index(self):
-        self.trycompletions('CREATE I', immediate='NDEX ')
+        self.trycompletions('CREATE IN', immediate='DEX ')
         self.trycompletions('CREATE INDEX ', choices=['<new_index_name>', 'IF', 'ON'])
         self.trycompletions('CREATE INDEX example ', immediate='ON ')
 
     def test_complete_in_drop_index(self):
-        self.trycompletions('DROP I', immediate='NDEX ')
+        self.trycompletions('DROP IN', immediate='DEX ')
```

Comment:
I think we should preserve this completion and follow up with your change here

```suggestion
        self.trycompletions('DROP I', choices=['DENTITY', 'NDEX '])
        self.trycompletions('DROP IN', immediate='DEX ')
```

Reply:
Since "self.trycompletions('DROP I', choices=['DENTITY', 'NDEX '])" test is making checks for both INDEX and IDENTITY, not just INDEX, should this stay in the  test_complete_in_drop_index(self) test? Or should these checks be moved elsewhere? Could also rename the test.

Reply:
Yeah, good point @shalnisundram .  I think we can keep as is.